### PR TITLE
[7.7.x] RHPAM-793 (JBPM-7032): Stunner - Missing Overview panel

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/RHPAM-793

This is the corollary of https://github.com/kiegroup/kie-wb-common/pull/1625 for 7.7.x

(cherry picked from commit 89356428442105f7ef3384c2c2ec90c76e02a629)

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins please retest this

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/editor/AbstractProjectDiagramEditorTest.java [hunk 1]
```diff
@@ -339,14 +398,59 @@ public void testMakeMenuBarWithoutUpdateProjectPermission() {
...
+
+        //Verify Overview widget was setup. It'd be nice to just verify(presenter).resetEditorPages(..) but it is protected
+        verify(overviewWidget).setContent(eq(overview),
+                                          eq(filePath));
+        verify(kieView).clear();
+        verify(kieView).addMainEditorPage(eq(view));
+        verify(kieView).addOverviewPage(eq(overviewWidget),
+                                        any(com.google.gwt.user.client.Command.class));
+
+        verify(presenter).onDiagramLoad();
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/editor/AbstractProjectDiagramEditorTest.java]: @manstis I can not build this PR, [AbstractProjectDiagramEditor](https://github.com/kiegroup/kie-wb-common/blob/7.7.x/kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/main/java/org/kie/workbench/common/stunner/project/client/editor/AbstractProjectDiagramEditor.java) misses `onDiagramLoad()`.

Comment (manstis) [reply kie-wb-common-stunner/kie-wb-common-stunner-extensions/kie-wb-common-stunner-project/kie-wb-common-stunner-project-client/src/test/java/org/kie/workbench/common/stunner/project/client/editor/AbstractProjectDiagramEditorTest.java]: Great thanks @jomarko :-) I was going to take a closer look this morning.

**Comment by manstis** [comment]
@jomarko Fixed (new commit). Let's wait for Jenkins.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Pull request merged**